src/components/toast/toast.stories.js

```diff
@@ -30,6 +30,7 @@ export const Basic = () => {
   };
 
   const children = text('children', 'My text');
+  const timeout = number('timeout', 0);
```

Comment:
The number addon throws an error when there is no value in the input, whilst that is a storybook issue it might be better to use `const timeout = text('timeout', '0');` instead as either string and number valid `timeout` types